client/src/components/CommentBody/CommentBody.css

```diff
@@ -0,0 +1,16 @@
+
+.CommentBody {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-start;
+    width: 737px;
```

Comment:
This probably shouldn't have an explicit width like this - we'll want it to be responsive!